data/common.yaml

```diff
@@ -0,0 +1,11 @@
+---
+python::version: '3'
```

Comment:
can you please keep the data in the init.pp instead common.yaml? That enables puppet-strings to pick it up and document it in the REFERENCE.md.

Reply:
at https://voxpupuli.org/docs/reviewing_pr/, we have:
> Are hiera yaml files added for data-in-modules? Ensure that the data is compatible with [hiera 5](https://puppet.com/docs/puppet/5.3/hiera_migrate.html#use-cases-for-upgrading-to-hiera-5). Static data that is equal across every supported operating system must stay in the init.pp, it shouldn’t be moved to a common.yaml due to [puppet-strings issue #250](https://github.com/puppetlabs/puppet-strings/issues/250).

Reply:
These parameters in `common.yaml` have overrides in other `data/**/*.yaml` and should be os-dependent.  They where previously documented as:

```
Default value: `$python::params::<foo>`
```

this line is not shown anymore.  If we put the value in `init.pp` instead of `common.yaml`, wrong default values would be documented for some operating systems.